README.md

```diff
@@ -131,7 +131,7 @@ To run the service from the command line you need to have Python version 3.12 or
 Launch the MQTT gateway with the mandatory parametersn and, optionally, the url to the MQTT broker.
 
 ```
-$ python ./mqtt_gateway.py -m tcp://my-broker-host:1883 -u <saic-user> -p <saic-pwd>
+$ python ./main.py -m tcp://my-broker-host:1883 -u <saic-user> -p <saic-pwd>
```

Comment:
We should probably add cd src &&

Reply:
Do you mean execution from the src folder?

./src/main.py